tests/migrations/test_commands.py

```diff
@@ -338,6 +339,127 @@ def test_showmigrations_plan_squashed(self):
+        "migrations.migrations_test_apps.mutate_state_b",
+        "migrations.migrations_test_apps.alter_fk.author_app",
+        "migrations.migrations_test_apps.alter_fk.book_app",
+    ])
+    def test_showmigrations_plan_multiple_app_labels(self):
+        """
+        Tests --plan output of `showmigrations`
+        with multiple app_labels provided.
+        """
+        # Multiple apps: author_app depends on book_app; mutate_state_b doesn't depend on other apps.
+        out = six.StringIO()
+        call_command("showmigrations", "mutate_state_b", "author_app", format='plan', stdout=out)
+        self.assertEqual(
+            "[ ]  author_app.0001_initial\n"
+            "[ ]  book_app.0001_initial\n"
+            "[ ]  author_app.0002_alter_id\n"
+            "[ ]  mutate_state_b.0001_initial\n"
+            "[ ]  mutate_state_b.0002_add_field\n",
+            out.getvalue()
+        )
+        # Multiple apps: args order shouldn't matter (the same result is expected as above).
+        out = six.StringIO()
+        call_command("showmigrations", "author_app", "mutate_state_b", format='plan', stdout=out)
+        self.assertEqual(
+            "[ ]  author_app.0001_initial\n"
+            "[ ]  book_app.0001_initial\n"
+            "[ ]  author_app.0002_alter_id\n"
+            "[ ]  mutate_state_b.0001_initial\n"
+            "[ ]  mutate_state_b.0002_add_field\n",
+            out.getvalue()
+        )
+
+    @override_settings(INSTALLED_APPS=[
+        "migrations.migrations_test_apps.mutate_state_a",
+        "migrations.migrations_test_apps.mutate_state_b",
+        "migrations.migrations_test_apps.alter_fk.author_app",
+        "migrations.migrations_test_apps.alter_fk.book_app",
+    ])
+    def test_showmigrations_plan_app_label_verbose(self):
+        """
+        Makes sure that `showmigrations --plan app_label -v 2`
+        output is verbose enough.
+        """
+        out = six.StringIO()
+        call_command("showmigrations", "author_app", "mutate_state_b", format='plan', stdout=out, verbosity=2)
+        self.assertEqual(
+            "[ ]  author_app.0001_initial\n"
+            "[ ]  book_app.0001_initial ... (author_app.0001_initial)\n"
+            "[ ]  author_app.0002_alter_id ... (author_app.0001_initial, book_app.0001_initial)\n"
+            "[ ]  mutate_state_b.0001_initial\n"
+            "[ ]  mutate_state_b.0002_add_field ... (mutate_state_b.0001_initial)\n",
+            out.getvalue()
+        )
+
+        # mutate_state_a app should be ignored even while it's in INSTALLED_APPS.
+        self.assertNotIn("mutate_state_a", out.getvalue())
+        self.assertIn("migrations.migrations_test_apps.mutate_state_a", settings.INSTALLED_APPS)
+
+    @override_settings(INSTALLED_APPS=["migrations.migrations_test_apps.unmigrated_app"])
+    def test_showmigrations_plan_app_label_error(self):
+        """
+        Makes sure that `showmigrations --plan app_label` raises error
```

Comment:
Chop "Makes sure" prefix in favor of stating the expected behavior as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/.